File: libgdx_drag_race/desktop/src/com/nopalsoft/dragracer/desktop/DesktopLauncher.java
package com.nopalsoft.dragracer.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.nopalsoft.dragracer.MainStreet;

public class DesktopLauncher {
    public static void main(String[] args) {
        LwjglApplicationConfiguration cfg = new LwjglApplicationConfiguration();
        cfg.title = "StreetSwipinRace";
        cfg.width = 480;
        cfg.height = 800;

        new LwjglApplication(new MainStreet(), cfg);
    }

}

File: libgdx_drag_race/desktop/src/com/nopalsoft/dragracer/desktop/DesktopLauncher.kt
package com.nopalsoft.dragracer.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.nopalsoft.dragracer.MainStreet


/**
 * warning: Don't remove args from this function.
 */
fun main(args: Array<String>) {

    val config = LwjglApplicationConfiguration()
    with(config) {
        title = "StreetSwipinRace"
        width = 480
        height = 800
    }

    LwjglApplication(MainStreet(), config)
}


